xml/ses_glossary.xml

```diff
@@ -25,14 +25,30 @@
     </para>
    </glossdef>
   </glossentry>
+  <glossentry><glossterm>&alertmanager;</glossterm>
+   <glossdef>
+    <para>
+     A single binary which handles alerts sent by the &prometheus; server and
+     notifies the end user.
+    </para>
+   </glossdef>
+  </glossentry>
+  <glossentry><glossterm>archive sync module</glossterm>
+   <glossdef>
+    <para>
+     Module that enables creating an &ogw; zone for keeping the history of S3
+     object versions.
+    </para>
+   </glossdef>
+  </glossentry>
   <glossentry><glossterm>Bucket</glossterm>
    <glossdef>
     <para>
      A point that aggregates other nodes into a hierarchy of physical
      locations.
     </para>
     <important>
-     <title>Do Not Mix with S3 Buckets</title>
+     <title>Do not mix with S3 buckets</title>
```

Comment:
By "mix" do we mean "confuse" here?

If the purpose of this para is to disambiguate S3 buckets from some other thing named "buckets" then I think maybe we should specify what that other thing is.

If we are disambiguating containers from (e.g.) Docker or k8s containers, then we should separate the entries for "bucket" and "container".

Reply:
I'm actually not sure, this was what you had previously?

Reply:
Sorry, I am sure: It's "do not confuse" - I don't know why I didn't read that the right way the first time. But I think that tells you that it's confusing as it is...